apps/designer/app/designer/features/workspace/workspace.tsx

```diff
@@ -1,7 +1,23 @@
 import { Box, Flex, Toaster } from "@webstudio-is/design-system";
+import { type Instance, type InstanceProps } from "@webstudio-is/react-sdk";
 import { useCanvasWidth, useZoom } from "~/designer/shared/nano-states";
 import { CanvasTools } from "./canvas-tools";
-import { type Publish } from "@webstudio-is/react-sdk";
+import { type Publish } from "~/shared/pubsub";
+
+declare module "~/shared/pubsub" {
```

Comment:
same here, location of this is kinda random, because those events are used all over the place